### ISSUE TITLE
Cloudflare R2 doesn't support x-amz-storage-class

### ISSUE THREAD

**Issue by dries-c** [gobackup/gobackup#182]
The R2 storage type doesn't work seem to work anymore.

```
NotImplemented: Header 'x-amz-storage-class' with value '' not implemented status code: 501, request id: , host id: 
```

**Comment by dries-c** [comment]
Duplicate new issue with a better explanation #183

**Comment by huacnlee** [comment]
Just released v2.5.1 for fix this.
